src/eradiate/scenes/biosphere/_rami_scenarios.py

```diff
@@ -0,0 +1,133 @@
+    ANISOTROPIC_BACKGROUND_OVERSTOREY_DENSE_BRF_MODEL_B = "HET21_DIS_D1B"
+    ANISOTROPIC_BACKGROUND_OVERSTOREY_DENSE_BRF_MODEL_C = "HET22_DIS_D1C"
+    TWO_LAYER_CANOPY_OVERSTORIES_SPARSE_UNDERSTORIES_SPARSE = "HET16_DIS_S2S"
+    TWO_LAYER_CANOPY_OVERSTORIES_MEDIUM_UNDERSTORIES_SPARSE = "HET17_DIS_M2S"
+    TWO_LAYER_CANOPY_OVERSTORIES_DENSE_UNDERSTORIES_SPARSE = "HET18_DIS_D2S"
+    TWO_LAYER_CANOPY_OVERSTORIES_SPARSE_UNDERSTORIES_DENSE = "HET26_DIS_S2D"
+    TWO_LAYER_CANOPY_OVERSTORIES_MEDIUM_UNDERSTORIES_DENSE = "HET27_DIS_M2D"
+    TWO_LAYER_CANOPY_OVERSTORIES_DENSE_UNDERSTORIES_DENSE = "HET28_DIS_D2D"
+    CONSTANT_SLOPE_DISTRIBUTION_SPARSE_INCLINATION_15 = "HET23_DIS_S15"
+    CONSTANT_SLOPE_DISTRIBUTION_DENSE_INCLINATION_15 = "HET24_DIS_D15"
+    CONSTANT_SLOPE_DISTRIBUTION_SPARSE_INCLINATION_30 = "HET33_DIS_S30"
+    CONSTANT_SLOPE_DISTRIBUTION_DENSE_INCLINATION_30 = "HET34_DIS_D30"
+
+
+class HomogeneousAbstractCanopies(Enum):
+    ANISOTROPIC_BACKGROUND_PLANOPHILE_A = "HOM23_DIS_P1A"
+    ANISOTROPIC_BACKGROUND_PLANOPHILE_B = "HOM24_DIS_P1B"
+    ANISOTROPIC_BACKGROUND_PLANOPHILE_C = "HOM25_DIS_P1C"
+    # ANISOTROPIC_BACKGROUND_ERECTOPHILE_A = "HOM33_DIS_E1A"
+    ANISOTROPIC_BACKGROUND_ERECTOPHILE_B = "HOM34_DIS_E1B"
+    ANISOTROPIC_BACKGROUND_ERECTOPHILE_C = "HOM35_DIS_E1C"
+    TWO_LAYER_CANOPY_ERECTOPHILE_SPARSE_PLANOPHILE_DENSE = "HOM26_DIS_EPD"
+    TWO_LAYER_CANOPY_ERECTOPHILE_SPARSE_PLANOPHILE_MEDIUM = "HOM27_DIS_EPM"
+    TWO_LAYER_CANOPY_ERECTOPHILE_SPARSE_PLANOPHILE_SPARSE = "HOM28_DIS_EPS"
+    TWO_LAYER_CANOPY_PLANOPHILE_SPARSE_ERECTOPHILE_DENSE = "HOM36_DIS_PED"
+    TWO_LAYER_CANOPY_PLANOPHILE_SPARSE_ERECTOPHILE_MEDIUM = "HOM37_DIS_PEM"
+    TWO_LAYER_CANOPY_PLANOPHILE_SPARSE_ERECTOPHILE_SPARSE = "HOM38_DIS_PES"
+    ADJACENT_CANOPIES_SPARSE_ERECTOPHILE_DENSE_PLANOPHILE = "HOM29_DIS_EM0"
+    ADJACENT_CANOPIES_MEDIUM_ERECTOPHILE_SPARSE_PLANOPHILE = "HOM30_DIS_ED0"
+
+
+class ScenarioVersion(Enum):
+    ORIGINAL = "original"
+    SIMPLIFIED = "simplified"
+
+
+def generate_name(
+    scenario_name: typing.Union[
+        ActualCanopies, HeterogeneousAbstractCanopies, HomogeneousAbstractCanopies
+    ],
+    version: ScenarioVersion = ScenarioVersion.ORIGINAL,
+) -> str:
+    """
+    Generate a name for a scenario based on its name and version.
+
+    Parameters
+    ----------
+    scenario_name : (ActualCanopies | HeterogeneousAbstractCanopies | HomogeneousAbstractCanopies)
+        The name of the scenario.
+    version : ScenarioVersion
+        The version of the scenario.
+
+    Returns
+    -------
+    str
+        The name of the scenario.
+    """
+    return (
+        f"{scenario_name.value}-{version.value}"
+        if version == ScenarioVersion.SIMPLIFIED
+        else scenario_name.value
+    )
+
+
+def load_rami_scenario(
+    scenario_name: typing.Union[
+        ActualCanopies, HeterogeneousAbstractCanopies, HomogeneousAbstractCanopies
+    ],
```

Comment:
From an interface point of view, I think it would be nice to accept strings here. Something like:

```python
load_rami_scenario("HET10_DIS_S1A")
```

This way, the user can easily get the scenario name from the RAMI docs and drop it here without having to import/access additional members or browse our documentation.

This shouldn't be too difficult to address even considering the multi-enum setup ([example here](https://gist.github.com/leroyvn/efbb1c9fd79f0e7def325e07b4337f18) with an enum value conflict check).